crates/extensions/tedge_config_manager/src/tests.rs

```diff
@@ -97,7 +97,7 @@ async fn new_config_manager_builder(
     };
 
     let mut mqtt_builder: SimpleMessageBoxBuilder<MqttMessage, MqttMessage> =
-        SimpleMessageBoxBuilder::new("MQTT", 5);
+        SimpleMessageBoxBuilder::new("MQTT", 20);
```

Comment:
My bad, I thought it was necessary to send all MQTT messages without blocking (concurrency test sends 5 config snapshot and 5 config update requests), but it must've been something else blocking me early on because it still works with capacity=5. Restored previous capacity.